src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -53,18 +53,14 @@ var validUTCDateTime = function (dateTimeComposition){
 };
 
 // check the string is valid
-var validString = function (dateTimeString) {
+var validDatetimeString = function (dateTimeString) {
     if (!dateTimeString || dateTimeString.toString() === '')
         return false;
-    return true;
-};
-
-// check the characters of string are digits
-var validDigitString = function (dateTimeString) {
+    // datetime format in the spec: YYYY[MM[DD[HH[MM[SS[.S[S[S[S]]]]]]]]][+/-ZZZZ],
```

Comment:
Is there at most four digits in milliseconds? It seems not consistent with the regex below.